From 12db9ddbfdc9c606b5421d9f6a6c157c671ccfe3 Mon Sep 17 00:00:00 2001
From: WingMan Kwok <w-kwok2@ti.com>
Date: Thu, 14 Dec 2017 18:41:00 -0500
Subject: [PATCH 140/140] ti_config_fragments/dra7_only.cfg: enable TI_PTP_BC

Include PTP boundary clock kernel driver by default on AM57xx
platforms.

Signed-off-by: WingMan Kwok <w-kwok2@ti.com>
---
 ti_config_fragments/dra7_only.cfg | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/ti_config_fragments/dra7_only.cfg b/ti_config_fragments/dra7_only.cfg
index db0d4ab..64b51b0 100644
--- a/ti_config_fragments/dra7_only.cfg
+++ b/ti_config_fragments/dra7_only.cfg
@@ -23,3 +23,5 @@ CONFIG_SOC_DRA7XX=y
 CONFIG_ARCH_KEYSTONE=n
 
 CONFIG_ARM_LPAE=y
+
+CONFIG_TI_PTP_BC=y
-- 
1.9.1

